src/main/java/de/berlin/lostberlin/service/mail/config/SendGridConfiguration.java

```diff
@@ -8,7 +8,7 @@
 @Configuration
 public class SendGridConfiguration {
 	
-	private static String API_KEY = "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";
+	private static String API_KEY = "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";
 	
```

Comment:
this may indeed be moved to some properties file as well.

Reply:
ok

Reply:
that's true, changed it